src/scripts/leaves.ts

```diff
@@ -114,22 +114,20 @@ class Leaves {
   }
 
   private loadLeaves() {
-    PIXI.loader
-      .add("leaf0", "assets/img/leaf0.png")
-      .add("leaf1", "assets/img/leaf1.png")
-      .add("leaf2", "assets/img/leaf2.png")
-      .add("leaf3", "assets/img/leaf3.png")
-      .load(() => this.createLeaves());
+    for (let i = 0; i < 13; ++i) {
+      PIXI.loader.add(`flake${i}`, `assets/img/flake${i}.png`)
+    }
+    PIXI.loader.load(() => this.createLeaves());
   }
 
   private createLeaves() {
     for(var i = 0; i < this.maximumLeaves; i++) {
-      var leafIdx = Math.round(Math.random()*50) % 4;
+      var flakeId = Math.floor(Math.random() * 13)
       var posX = Math.random() * this.renderer.view.width;
       var posY = Math.random() * this.renderer.view.height;
       var radius = this.minSize + Math.random() * this.maxSize + 1;
```

Comment:
No idea, I haven't looked into that yet. I did notice they seemed to fall too fast for my taste.